R/SSplotEnsemble.R

```diff
@@ -1,73 +1,42 @@
 #' Model ensemble plots
 #'
-#' Plots model ensembles and forecasts with uncertaity represented by MVLN or MCMC posteriors
+#' Plots model ensembles and forecasts with uncertainty represented by MVLN or
+#' MCMC posteriors
 #'
-#' @param kb SSdeltaMVLN $kb type output
-#' @param subplots option to "Bratio","Fvalue","SSB", "F", "Recr","Catch"
+#' @param kb `kb` type output created by `SSdeltaMVLN()`
+#' @param subplots vector to create subplots with these options:
+#' \itemize{
+#'  \item `"stock"` Fish Population
```

Comment:
I would say stock is 'Stock biomass relative to reference point (B/BMSY)' and harvest 'Harvest rate relative to reference point (F/FMSY)'